goreq.go

```diff
@@ -436,7 +437,9 @@ func (r Request) NewRequest() (*http.Request, error) {
 	}
 
 	var bodyReader io.Reader
-	if b != nil && r.Compression != nil {
+	if r.BodyReader != nil {
```

Comment:
You are right. I didn't see previously. Was my mistake don't read all code. I can send my request with something like:

```
res, err := goreq.Request{
    Method: "POST",
    Uri: "http://www.google.com",
    ContentType: "application/x-www-form-urlencoded",
    Body: params.Encode(),
}.Do()
```
An apology.